Simulator/Core/Core.cpp

```diff
@@ -55,15 +55,15 @@ bool Core::parseCommandLine(string executableName, string cmdLineArguments)
    if ((cl.addParam("configfile", 'c', ParamContainer::filename, "parameter configuration filepath")
         != ParamContainer::errOk)
 #if defined(USE_GPU)
-       || (cl.addParam("device", 'd', ParamContainer::regular, "CUDA device id")
+       || (cl.addParam("device", 'g', ParamContainer::regular, "CUDA device id")
```

Comment:
To make the hint explain more clearly why it's `-g`, let's make it `"CUDA GPU device ID"`